lib/buildsystems/cmake.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class CMake < Package
+
+  def self.cmake_options(options = '')
+    return (@cmake_options = options if options)
+  end
+
+  def self.check?(bool = true)
+    return (@check = bool)
+  end
+
+  def self.build
+    system "cmake -B builddir -G Ninja #{CREW_CMAKE_OPTIONS} #{@cmake_options}"
```

Comment:
How would you override `CREW_CMAKE_OPTIONS`?

Reply:
My initial thought is that if you want to deviate from the `buildsystems` setup, then just don't use them, and then the package is just like in status quo ante...

Changes which are used often enough can be migrated as an option into `buildsystems`...

Reply:
Made an issue for dealing with this later: https://github.com/chromebrew/chromebrew/issues/8434 , since this isn't necessary to get the larger functionality in place.